### PR TITLE
Renames all typepaths + names of implants to biochips

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
They're biochips, not implants goddamit!

EDIT: Due to me misunderstanding contra, this PR is now changing literally every typepath that should have said "bio-chips" or similar.

## Why It's Good For The Game
We switched over the naming a loooong time ago, this was forgotten.

## Testing
Spawned in chip implanters

## Changelog
:cl:
fix: The Bio-chip uplink category is now properly named so
/:cl:

### CODE REVIEW

**Review by Contrabang — CHANGES_REQUESTED** [review]
Can you change the comments and the typepaths too? I think we dont want to leave this around as debt if we're changing this

**Comment by DGamerL** [comment]
> Can you change the comments and the typepaths too? I think we dont want to leave this around as debt if we're changing this

Sounds good to me, gimme a bit to change it all

File: code/datums/mind.dm [hunk 1]
```diff
@@ -798,14 +798,14 @@
 
 		switch(href_list["implant"])
 			if("remove")
-				for(var/obj/item/implant/mindshield/I in H.contents)
+				for(var/obj/item/bio_chip/mindshield/I in H.contents)
 					if(I && I.implanted)
 						qdel(I)
 				to_chat(H, "<span class='notice'><Font size =3><B>Your mindshield bio-chip has been deactivated.</B></FONT></span>")
```
Comment (Burzah) [inline code/datums/mind.dm:804]:
```suggestion
				to_chat(H, "<span class='notice'><font size =3><b>Your mindshield bio-chip has been deactivated.</b></font></span>")
```
While we are here.

File: code/__HELPERS/unsorted.dm [hunk 2]
```diff
@@ -1733,7 +1733,7 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			/obj/item/grenade/clusterbuster = "CLUSTERBUSTER",
 			/obj/item/grenade = "GRENADE",
 			/obj/item/gun = "GUN",
-			/obj/item/implant = "IMPLANT",
+			/obj/item/bio_chip = "IMPLANT",
```
Comment (Contrabang) [inline code/__HELPERS/unsorted.dm:1736]:
```suggestion
			/obj/item/bio_chip = "BIO_CHIP",
```

File: code/modules/research/designs/medical_designs.dm [hunk 3]
```diff
@@ -544,14 +544,14 @@
...
 	desc = "A sterile automatic bio-chip injector."
 	id = "implanter"
 	req_tech = list("materials" = 2, "biotech" = 3, "programming" = 2)
 	build_type = PROTOLATHE
 	materials = list(MAT_METAL = 600, MAT_GLASS = 200)
-	build_path = /obj/item/implanter
+	build_path = /obj/item/bio_chip_implanter
 	category = list("Medical")
 
 /datum/design/implantcase
```
Comment (Contrabang) [inline code/modules/research/designs/medical_designs.dm:557]:
```suggestion
/datum/design/bio_chip_case
```

File: code/modules/research/designs/medical_designs.dm [hunk 4]
```diff
@@ -561,7 +561,7 @@
 	req_tech = list("biotech" = 2)
 	build_type = PROTOLATHE
 	materials = list(MAT_GLASS = 500)
-	build_path = /obj/item/implantcase
+	build_path = /obj/item/bio_chip_case
 	category = list("Medical")
 
 /datum/design/implant_chem
```
Comment (Contrabang) [inline code/modules/research/designs/medical_designs.dm:567]:
```suggestion
/datum/design/bio_chip_chem
```

File: code/modules/research/designs/medical_designs.dm [hunk 5]
```diff
@@ -571,7 +571,7 @@
 	req_tech = list("materials" = 3, "biotech" = 5)
 	build_type = PROTOLATHE
 	materials = list(MAT_GLASS = 700)
-	build_path = /obj/item/implantcase/chem
+	build_path = /obj/item/bio_chip_case/chem
 	category = list("Medical")
 
 /datum/design/implant_sadtrombone
```
Comment (Contrabang) [inline code/modules/research/designs/medical_designs.dm:577]:
```suggestion
/datum/design/bio_chip_sad_trombone
```

File: code/modules/research/designs/medical_designs.dm [hunk 6]
```diff
@@ -581,7 +581,7 @@
 	req_tech = list("materials" = 3, "biotech" = 5)
 	build_type = PROTOLATHE
 	materials = list(MAT_GLASS = 500, MAT_BANANIUM = 500)
-	build_path = /obj/item/implantcase/sad_trombone
+	build_path = /obj/item/bio_chip_case/sad_trombone
 	category = list("Medical")
 
 /datum/design/implant_tracking
```
Comment (Contrabang) [inline code/modules/research/designs/medical_designs.dm:587]:
```suggestion
/datum/design/bio_chip_tracking
```

File: code/modules/response_team/ert_outfits.dm [hunk 7]
```diff
@@ -85,8 +85,8 @@
 	)
 
 	implants = list(
```
Comment (Contrabang) [inline code/modules/response_team/ert_outfits.dm:87]: Maybe rename this var thats on outfits to `bio_chips`

File: code/modules/surgery/implant_removal.dm [hunk 8]
```diff
@@ -31,7 +31,7 @@
 	allowed_tools = list(TOOL_HEMOSTAT = 100, TOOL_CROWBAR = 65)
 	time = 6.4 SECONDS
 	repeatable = TRUE
-	var/obj/item/implant/I = null
+	var/obj/item/bio_chip/I = null
 	var/max_times_to_check = 5
 
 /datum/surgery_step/extract_implant/begin_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
```
Comment (Contrabang) [inline code/modules/surgery/implant_removal.dm:37]:
```suggestion
/datum/surgery_step/extract_bio_chip/begin_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
image
(insert sisyphus music here)

File: code/modules/response_team/ert_outfits.dm (hunk 7 above)
Comment (DGamerL) [reply code/modules/response_team/ert_outfits.dm:87]: Will do!

**Review by DGamerL — COMMENTED** [review]
I'll have to rename a few files aswell, will do in a bit

**Comment by McRamon** [comment]
Are we going to call implanters biochipers now?

File: paradise.dme [hunk 9]
```diff
@@ -1115,30 +1115,30 @@
...
+#include "code\game\objects\items\weapons\implants\bio_chip_mindshield.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_pad.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_sad_trombone.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_stealth.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_storage.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_supercharge.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_tracking.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_traitor.dm"
+#include "code\game\objects\items\weapons\implants\bio_chip_uplink.dm"
+#include "code\game\objects\items\weapons\implants\bio_chipper.dm"
```
Comment (Contrabang) [inline paradise.dme:1141]: This folder should probably be renamed too

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Contrabang — CHANGES_REQUESTED** [review]
/datum/uplink_item/bio_chips/krav_implant
	name = "Krav Maga Implant"
	
name needs to be changed

signals.dm should probably also have its signal defines updated

/obj/item/storage/hidden/implant path should be updated

GLOB.tracked_implants should be renamed

**Comment by S34NW** [comment]
> ## Why It's Good For The Game
> 
> We switched over the naming a loooong time ago, this was forgotten.

It was not forgotten it was just done at a time where map edits were hell :)

**Review by Contrabang — CHANGES_REQUESTED** [review]
one last thing

File: code/game/objects/items/weapons/dice.dm [hunk 10]
```diff
@@ -261,7 +261,9 @@
 				/obj/item/borg/upgrade/modkit/indoors,
 				/obj/item/storage/box/syndie_kit/chameleon,
 				/obj/item/mod/control/pre_equipped/traitor,
-				/obj/item/implanter/storage,
+				/obj/item/bio_chip_implanter/storage,
+				/obj/item/mod/control/pre_equipped/traitor,
+				/obj/item/bio_chip_implanter/storage,
```
[the remark this answers is not in the document]
Comment (DGamerL) [reply code/game/objects/items/weapons/dice.dm:266]: Yep, conflict resolving caused it, good catch!

File: code/game/objects/items/weapons/dice.dm [hunk 11]
```diff
@@ -261,7 +261,8 @@
 				/obj/item/borg/upgrade/modkit/indoors,
 				/obj/item/storage/box/syndie_kit/chameleon,
 				/obj/item/mod/control/pre_equipped/traitor,
-				/obj/item/implanter/storage,
+				/obj/item/bio_chip_implanter/storage,
+				/obj/item/mod/control/pre_equipped/traitor,
```
Comment (Contrabang) [inline code/game/objects/items/weapons/dice.dm:265]: This one too
```suggestion
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Pull request merged**